pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -151,13 +155,14 @@ pub mod pallet {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
-		Todo,
+		PoolCreated(T::PoolId),
 	}
 
-	// Errors inform users that something went wrong.
 	#[pallet::error]
 	pub enum Error<T> {
-		Todo,
+		CurrenciesNumber,
```

Comment:
Can you perhaps use a more descriptive name for this error?

Reply:
Renamed in: [e401cef](https://github.com/KILTprotocol/kilt-node/pull/768/commits/e401cef05fe348907c08b9894c1e7c5e1cfe5740)